airflow/www/templates/airflow/dagrun_details.html

```diff
@@ -0,0 +1,44 @@
+{#
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+#}
+
+{% extends "airflow/dag.html" %}
+{% block page_title %}{{ dag_id }} - {{ run_id }} - DAG Run details - {{ appbuilder.app_name }}{% endblock %}
+
+{% block content %}
+  {{ super() }}
+  <hr>
+  <br>
+  <h4>
+    <span class="text-muted">DAG Run details:</span> <span>{{ dag_id }}</span>
+    <span class="text-muted">run id</span> <span>{{ run_id }}</span>
+    <span class="text-muted">at</span> <time datetime="{{ execution_date }}">{{ execution_date }}</time>
+  </h4>
+  <table class="table table-striped table-bordered">
+    <tr>
+      <th>Attribute</th>
+      <th>Value</th>
+    </tr>
+    {% for attribute, value in dagrun_attributes %}
+      <tr>
+        <td class="text-nowrap">{{ attribute }}</td>
+        <td>{{ value }}</td>
```

Comment:
Can you confirm that the dates here update when you change the app's timezone?

Reply:
Currently only this date updates:
![image](https://user-images.githubusercontent.com/6249654/142624260-ee8022c4-af63-42d7-8e29-817c6d2d83e0.png)

The other dates include timezone offsets, shall I make them all update when changing the timezone?

Reply:
Yes, I believe we'd want all of them to update. I think we have another spot where we check if an attribute value is a date and wrap it in a `<timezone>`

Reply:
Done. By default, the milliseconds are not displayed so it now looks like:
![image](https://user-images.githubusercontent.com/6249654/142653530-0af8b0ec-fc8b-4c64-bd6f-8bedd0ceab85.png)